simpleclient_spring_boot/src/main/java/io/prometheus/client/spring/boot/EnablePrometheusEndpoint.java

```diff
@@ -0,0 +1,34 @@
+package io.prometheus.client.spring.boot;
+
+import org.springframework.context.annotation.Import;
+
+import java.lang.annotation.Documented;
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
+
+/**
+ * Enable an endpoint that exposes Prometheus metrics from its default collector.
+ * <p>
+ * Usage:
+ * <br>Just add this annotation to your main Spring Boot Application.
+ * <p>
+ * Configuration:
+ * <br>This endpoint is configurable through the following environment variables:
+ * <ul>
+ * <li>{@code endpoints.prometheus.id} (default: "prometheus")</li>
```

Comment:
I'm not sure `endpoints.prometheus.id` is a valid environment variable. Are periods allowed in environment variable names? It's usually underscores.